Sources/SATSCore/Extensions/UIKit/UILabel-Utilities.swift

```diff
@@ -0,0 +1,14 @@
+import UIKit
+
+extension UILabel {
+    func setText(text: String?, withLineHeightMultiple lineHeight: CGFloat) {
+        guard let text = text else { return }
+        let style = NSMutableParagraphStyle()
+        style.lineHeightMultiple = lineHeight
+        let attributeString = NSMutableAttributedString(
+            string: text,
+            attributes: [NSAttributedString.Key.paragraphStyle: style]
```

Comment:
I think it's possible to write this like `[.paragraphStyle: style]` if the keys of the dictionary are typed